.github/workflows/rc.yml

```diff
@@ -91,3 +92,20 @@ jobs:
             git checkout -b $branch $commit_sha
             git push -f origin $branch
           fi
+
+      - name: Update the technical docs link in the nav.adoc file
+        run: |
+          major_minor_version=$(echo "${{ steps.get_version.outputs.version }}" | cut -d'.' -f1,2)
+          echo "Major and minor version extracted: $major_minor_version"
+          sed -i '' -E "s|(https://)[^/]*openzeppelin-relayer\.netlify\.app|\1release-v${major_minor_version}--openzeppelin-relayer.netlify.app|g" docs/modules/ROOT/nav.adoc
+
```

Comment:
Why not add this in "on push" to release-v* branch add condition to check if it's release branch version prefix url for netlify if not update it? This workflow we can keep just strictly to create a release branch